test/executable/executableTests.ts

```diff
@@ -349,15 +347,21 @@ describe("Executable", function(this: Mocha.ISuiteCallbackContext) {
                 });
         });
 
-        it("can handle multiple '--exclude' globs", (done) => {
+        it("can apply fixes from multiple rules", function(done) {
+            this.slow(4000);
+            this.timeout(6000);
+            fs.writeFileSync("test/files/project-multiple-fixes/testfile.test.ts",
+                             fs.readFileSync("test/files/project-multiple-fixes/before.test.ts", "utf-8"));
```

Comment:
we could probably use some better infrastructure so all this juggling isn't required. but this is fine for now.